addons/payment_stripe/models/payment_acquirer.py

```diff
@@ -27,6 +31,86 @@ class PaymentAcquirer(models.Model):
              "authenticate the messages sent from Stripe to Odoo.",
         groups='base.group_system')
 
+    # === ACTION METHODS === #
+
+    def action_stripe_connect_account(self, menu_id=False):
+        """Create a Stripe Connect account and redirect the user to the next onboarding step.
+
+        If the acquirer is already enabled, close the current window. Otherwise, generate a Stripe
+        Connect onboarding link and redirect the user to it. If the link generation failed, redirect
+        the user to the acquirer form.
+
+        Note: self.ensure_one()
+
+        :param str onboarding: The module from which the onboarding was initiated
+        :return: The next step action
+        :rtype: dict
+        """
+        self.ensure_one()
+
+        if self.state == 'enabled':
+            action = {'type': 'ir.actions.act_window_close'}
+
+        # Account creation
+        connected_account = self._stripe_fetch_or_create_connected_account()
+
+        # Link generation
+        if menu_id is False:
+            menu_id = self.env.ref('payment.payment_acquirer_menu').id
+        account_link_url = self._stripe_account_link_url(connected_account['id'], menu_id)
+        if account_link_url:
+            action = {
+                'type': 'ir.actions.act_url',
+                'url': account_link_url,
+                'target': 'self',
+            }
+        else:
+            action = self.env['ir.actions.actions']._for_xml_id('payment.action_payment_acquirer')
+            action.update(views=[[False, 'form']], res_id=self.id)
```

Comment:
The first value of `action` is never used anywhere.
```suggestion
        if self.state == 'enabled':
            action = {'type': 'ir.actions.act_window_close'}
        else:
            # Account creation
            connected_account = self._stripe_fetch_or_create_connected_account()
    
            # Link generation
            if menu_id is False:
                menu_id = self.env.ref('payment.payment_acquirer_menu').id
            account_link_url = self._stripe_account_link_url(connected_account['id'], menu_id)
            if account_link_url:
                action = {
                    'type': 'ir.actions.act_url',
                    'url': account_link_url,
                    'target': 'self',
                }
            else:
                action = self.env['ir.actions.actions']._for_xml_id(
                    'payment.action_payment_acquirer'
                )
                action.update(views=[[False, 'form']], res_id=self.id)
```